src/lib.rs

```diff
@@ -60,6 +60,8 @@ pub use crate::pac as stm32;
 
 pub mod traits;
 
+#[cfg(feature = "stm32l4x3")]
```

Comment:
It also needs updating here to support the rest of the MCUs.

Reply:
I added that to reflect what I had tested. I'm sure it compiles and works for other targets too (at least some of them).